imports/plugins/core/core/server/no-meteor/schemas/packageSettings.graphql

```diff
@@ -0,0 +1,15 @@
+type MailData {
+  user: String!
+  password: String!
+}
+
+type CoreSettingsData {
+  mail: MailData!
+}
+
+union PackageSettingsData = CoreSettingsData
```

Comment:
Since there's a standard where everything under `public` is public, it might be better to make this an interface and make public a union. If that ends up being too weird, then what you have is fine.

```js
interface PackageSettingsData {
  "Settings that should be public"
  public: PackageSettingsPublicData
}

union PackageSettingsPublicData = JSONObject
```